languageServer/src/regexes.ts

```diff
@@ -1,3 +1,4 @@
 export const variableTemplates = /\${([^}]+)}/g
 export const filePaths = /[\d/A-Za-z]+\.[A-Za-z]+/g
 export const alphadecimalWords = /[\dA-Za-z]+/g
+export const propertyPathLike = /[\d.A-Z[\]a-z]+/g
```

Comment:
Any differences between this file and languageServer/src/languageHelpers/regexes.ts? They should be merged if so. Either way we should only have one file only for regexes.

Reply:
Forgot about it, will delete it 👍